content/articles/top-container-orchestration-tools/index.md

```diff
@@ -0,0 +1,137 @@
+
+If you have applications to develop, deploy and run, containers come in handy. With containers, applications are moved seamlessly between servers. Therefore, you need a management system that automatically runs containers, shuts them down when needed, and controls how containers access resources. That is where orchestration tools come in.
+
+Container orchestration tools enable developers to manage and automate containerized application tasks. These tasks include linking containers, running instances, provisioning hosts, and rescheduling failed containers.
+
+Container orchestration tools help in extending the lifecycle of applications, ease deployment, and manage application configurations. These tools also improve container security by setting limits while accessing containers.
+
+This article will introduce you to container orchestration tools. Also, it will give you an overview of container orchestration and why it is important.
+
```

Comment:
You could add a `Prerequisite` section mentioning the prerequisite knowledge that the reader must have, to understand this article better.

You could link to other Section articles related to Dockers and Containers. That would help beginners.